reporter-web-app/src/models/MetaData.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+class MetaData {
```

Comment:
Instead of just `MetaData` how about `OrtJob` or `OrtRun` to make the name less generic?

Reply:
@fviernau  We want to add more data into `MetaData` in follow up PRs such as the ORT version, as requested in https://github.com/oss-review-toolkit/ort/issues/3901

Reply:
hm, I see - my proposal doesn't really improve the situation then. I'm good with just MetaData then.